PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -32,7 +32,8 @@ using namespace o2::framework::expressions;
 
 enum DecayChannel { DplusToPiKPi = 0,
                     DsToKKPi,
-                    DsToPiKK };
+                    DsToPiKK,
+                    D0ToPiK };
```

Comment:
Please use the centrality estimator `enum` defined in https://github.com/AliceO2Group/O2Physics/blob/7bc403b421e84a0c709ce7af0185d8aa40f0b556/PWGHF/DataModel/CandidateReconstructionTables.h#L65

Reply:
Please fix the name of `qvecEstimator` -> `QVecEstimator`.